### DISCUSSION TITLE
NavBar menu for small screens

### DISCUSSION THREAD

**Discussion by JLambertazzo** [sButtons/sbuttons#1299] [Ideas]
I noticed some weird behaviour on the navbar when viewing on a small screen. I think it might be helpful to have a dropdown menu which shows on smaller screens.

**Comment by shahednasser** [comment]
Hello @JLambertazzo can you please specify the weird behavior? I don't mind the dropdown menu idea

**Reply by JLambertazzo** [reply]
![Capture](https://user-images.githubusercontent.com/42924425/108413100-96c12300-71f8-11eb-9e89-a56de1a7c001.PNG)
On screens around 490px and again under 380px the navbar is too wide and either creates some extra whitespace on the side or wraps onto a new line depending on the device.

**Reply by shahednasser** [reply]
Yes I noticed that and was planning on opening an issue with it, but totally forgot, to be honest.

I'll open an issue with it now. Please comment on there if you want to be assigned.
